core/services/telemetry/manager_test.go

```diff
@@ -304,7 +305,7 @@ func TestLegacyMode(t *testing.T) {
 	require.Equal(t, true, tm.legacyMode)
 	require.Len(t, tm.endpoints, 1)
 
-	clientSent := make([]synchronization.TelemPayload, 0)
+	var clientSent []synchronization.TelemPayload
```

Comment:
The linter understands the length check down below when we use the simpler declaration style :shrug: